### PR TITLE
Add Google Storage Lock

### PR DESCRIPTION
**Pull request by veziak**

### CODE REVIEW

File: pkg/gcp/gcp_lock.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+)
+
+type GoogleStorageLock struct {
+	Client *storage.Client
+	Bucket *storage.BucketHandle
+	ctx    context.Context
+}
+
+type Lock interface {
+	Lock(transactionId int, resource string) (bool, error)
```
Comment (Spartakovic) [inline pkg/gcp/gcp_lock.go]: we should use existing interface

Comment (veziak) [reply pkg/gcp/gcp_lock.go]: done

File: cmd/digger/main.go [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
 	}
-	sess := session.Must(session.NewSession())
-	dynamoDb := dynamodb.New(sess)
-	dynamoDbLock := aws.DynamoDbLock{DynamoDb: dynamoDb}
+	println("Digger config read successfully")
+
+	lock, err := utils.GetLock()
+	if err != nil {
+		fmt.Printf("Failed to create lock. %s\n", err)
+		os.Exit(2)
```
Comment (Spartakovic) [inline cmd/digger/main.go:24]: why do we want different exit codes?

Comment (veziak) [reply cmd/digger/main.go:24]: it was convenient when no error message has been displayed, not sure myself if we need to leave it this way or return to exit code=1

**Review by Spartakovic — APPROVED** [review]

**Pull request merged**